e2e/miniconstellation/main.tf

```diff
@@ -135,7 +135,7 @@ resource "azurerm_linux_virtual_machine" "main" {
     publisher = "Canonical"
     offer     = "0001-com-ubuntu-server-jammy-daily"
     sku       = "22_04-daily-lts"
-    version   = "latest"
+    version   = "22.04.202305020" // latest versions have permission error : Error: error creating libvirt domain: internal error: Process exited prior to exec: libvirt: QEMU Driver error : failed to umount devfs on /dev: Permission denied
```

Comment:
Is this still true with the downgrade qemu version?

Reply:
no it's not, good catch. i'm reverting it